datafusion/core/src/datasource/physical_plan/parquet/mod.rs

```diff
@@ -120,33 +121,41 @@ impl ParquetExec {
             MetricBuilder::new(&metrics).global_counter("num_predicate_creation_errors");
 
         let file_schema = &base_config.file_schema;
-        let pruning_predicate = predicate
-            .clone()
-            .and_then(|predicate_expr| {
-                match PruningPredicate::try_new(predicate_expr, file_schema.clone()) {
+        let pruning_predicate = if config.pruning {
```

Comment:
It seems that whether this pushdown is enabled is decided by `ParquetFormat.enable_pruning || config.pruning`.

`ParquetFormat.enable_pruning` is used in `ParquetFormat::create_physical_plan`.

Reply:
I would add that